.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
```

Comment:
Me too -- I think we need an stand-alone issue about the testing matrix / configuration.

And I'm lost on what I'm supposed to be +0.5 on :-) 

and I like @rsignell-usgs 's simple example -- seems straightforward. though is there even any need for the environment? not a big deal, but the root miniconda environment will be tossed out anyway when this is done.